SPECS/xcb-proto/xcb-proto.spec

```diff
@@ -2,20 +2,16 @@
 %define debug_package %{nil}
 Summary:        XCB protocol descriptions
 Name:           xcb-proto
-Version:        1.13
-Release:        16%{?dist}
+Version:        1.16.0
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://xcb.freedesktop.org/
-Source0:        https://xcb.freedesktop.org/dist/%{name}-%{version}.tar.bz2
+Source0:        https://xorg.freedesktop.org/archive/individual/proto/%{name}-%{version}.tar.gz
```

Comment:
URL update is not needed. Additionally, adding .0 seems like a new addition on this URL for the tarball versions. Not sure, if we should deviate?